### PR TITLE
Update windows_startup.tmpl

### PR DESCRIPTION
**Pull request by glattercj**

### CODE REVIEW

**Comment by glattercj** [comment]
Bump

**Comment by activeshadow** [comment]
@glattercj hmmm... seems like you're in a hurry for this. In that case, I think I'll review it a few more times just to make sure it's all good. :joy:

File: phenix/src/go/tmpl/templates/windows_startup.tmpl [hunk 1]
```diff
@@ -1,108 +1,89 @@
...
-$wmi.SetGateways('{{ $iface.Gateway }}', 1) | Out-Null
+    $status = $wmi.EnableStatic('{{ $iface.Address }}', '{{ $mask }}')
+    {{ end }}
+    Start-Sleep -Milliseconds 500
+} While ($status.ReturnValue -eq 2147786788)
+    {{ if ne $iface.Gateway "" }}
+      {{ if gt $length 1 }}
+    $wmi[{{ $idx }}].SetGateways('{{ $iface.Gateway }}', 1) | Out-Null
+      {{ else }}
+    $wmi.SetGateways('{{ $iface.Gateway }}', 1) | Out-Null
```
Comment (activeshadow) [inline phenix/src/go/tmpl/templates/windows_startup.tmpl:19]: Why did you indent these two `$wmi` calls? They're not in the a loop or if statement (it doesn't look like)...

File: phenix/src/go/tmpl/templates/windows_startup.tmpl [hunk 2]
```diff
@@ -1,108 +1,89 @@
...
+      {{ end }}
```
Comment (activeshadow) [inline phenix/src/go/tmpl/templates/windows_startup.tmpl:20]: Looks like you're indenting everything by 4 spaces now. Shouldn't this `if/else/end` template block be indented a few more spaces?

**Review by activeshadow — COMMENTED** [review]
@glattercj just a couple of inconsistencies I noticed.

File: phenix/src/go/tmpl/templates/windows_startup.tmpl (hunk 1 above)
Comment (glattercj) [reply phenix/src/go/tmpl/templates/windows_startup.tmpl]: No idea :| How's this?

File: phenix/src/go/tmpl/templates/windows_startup.tmpl (hunk 2 above)
Comment (glattercj) [reply phenix/src/go/tmpl/templates/windows_startup.tmpl]: Updated.

**Pull request merged**